test/backend.go

```diff
@@ -75,21 +88,6 @@ func NewMockBackend() *MockBackend {
 	}
 }
 
-func (mb *MockBackend) WrittenMsgs() []courier.Msg                   { return mb.writtenMsgs }
-func (mb *MockBackend) WrittenMsgStatuses() []courier.MsgStatus      { return mb.writtenMsgStatuses }
-func (mb *MockBackend) WrittenChannelEvents() []courier.ChannelEvent { return mb.writtenChannelEvents }
-func (mb *MockBackend) WrittenChannelLogs() []*courier.ChannelLog    { return mb.writtenChannelLogs }
-
-// LastContactName returns the contact name set on the last msg or channel event written
-func (mb *MockBackend) LastContactName() string {
-	return mb.lastContactName
-}
-
-// MockMedia adds the given media to the mocked backend
-func (mb *MockBackend) MockMedia(media courier.Media) {
-	mb.media[media.URL()] = media
-}
```

Comment:
moving these methods which aren't actually part of the Backend interface down to the bottom in their own section